man/g.report.part5.Rd

```diff
@@ -48,6 +50,16 @@
     Inclusion criteria for number of valid hours, either
     as expressed as a ratio of 1 or as the number of hours in a 24 hour day.
   }
+  \item{segmentWEARcrit.part5}{
+    Inclusion criteria for number of valid hours in the segments defined with
+    qwindow in part 5, expressed as a ratio of 1.
+  }
+  \item{segmentDAYSPTcrit.part5}{
+    Numeric vector of length 2 with the inclusion criteria for the proportion
+    of the segment that should be classified as day (awake) and spt (sleep period 
+    time) to be considered valid. Usually, one of the two numbers is 0, and the other
+    defines the proportion of the segment that should be classified as day or spt.
```

Comment:
- I think ", and the other defines the proportion of the segment that should be classified as day or spt" can be omitted as it repeats the information that is mentioned in the beginning of the explanation.
- Text does not refer to `qwindow` so it may be unclear what segment this is referring to?